oomph-actions/vars.yml

```diff
@@ -10,29 +10,50 @@ run_npm_install: no
 
 # Use Lando for build process
 use_lando: yes
-run_lando_composer_install: yes
+run_lando_composer_install: no
 # Disable the standard Oomph lando npm install.
 run_lando_npm_install: no
 
 # Use pre-installed Docker EE
 lando_install_prereqs: no
 lando_docker_ce: no
 
+# Append the hooks directory to the deploy files variable.
+drupal_deploy_files:
+  - config
```

Comment:
Do we need the config folder if we are managing via the installation profile, features, and then site specific? Wondering if we can drop this and let the sites use the default configuration folder?